src/components/Onboarding/index.js

```diff
@@ -70,9 +70,9 @@ const Onboarding = ({ open, step, state, ...props }) => {
           onboardingProps.setState(OnboardingStates.SETUP_SINGLE_WALLET)
         }
       />
-      {state === OnboardingStates.SETUP_LINKED_WALLET ? (
+      {open && state === OnboardingStates.SETUP_LINKED_WALLET ? (
```

Comment:
1 - Adding this prevents the component from mounting when the page is loaded, but the bug would still occur when the user re-enters onboarding